app/controllers/draws_controller.rb

```diff
@@ -199,14 +198,21 @@ def calculate_ungrouped_students_metrics
     @ungrouped_students.delete('off_campus')
   end
 
-  def prepare_suites_edit_data # rubocop:disable AbcSize
+  def prepare_suites_edit_data # rubocop:disable AbcSize, MethodLength
```

Comment:
this makes me feel like we need a whole separate `DrawSuitesController` but leaving it alone for now